change/office-ui-fabric-react-2019-12-12-10-18-09-toggle-div-in-button.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```

Comment:
Concerns with making this a `minor` instead? I have seen CSS that targets the `div` inside a `Button`, and this change will require changes by those users.

Reply:
sounds legit